core/src/main/java/org/fao/geonet/kernel/AccessManager.java

```diff
@@ -123,11 +121,6 @@
             } else {
                 results = new HashSet<>(operations);
             }
-
```

Comment:
The removed code here appears to be doing something, adding additional operations if they are authenticated, is this logic used for anything?

If the service context is not used here, can it be removed as a Parmeter?

aside: I lack some context for this one, can we make a guess on the method purpose to write up as a javadoc?

Reply:
Profile cannot be Reviewer and Editor at the same time: profile is the "higher" role a user belong to. Seems there was a "wrong" merge a few years ago which added the last condition and made the whole unrealisable. This said, imo, "editor should neither earn view rights on every md", and it seems it doesn't harms for years/month.

(imo, purpose of the method should be inferred from code or test reading, if I had to choose between writing a javadoc (and maintain it) or writing a test or writing clear code, I would choose the lasts...)